.github/workflows/release.yaml

```diff
@@ -24,38 +24,32 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Submodule Tag
-        if: ${{ github.event.inputs.submodule_tag }}
+      - name: Check submodule
```

Comment:
Please be more specific that we are looking for a Go submodule: `Check Go submodule`